application/models/SurveysGroupsettings.php

```diff
@@ -395,7 +395,12 @@ protected static function translateOptionLabels($instance, $attribute, $value)
         } elseif ($attribute == 'owner_id' && $value != -1) {
             $instance->oOptions->owner = "";
             $instance->oOptions->ownerLabel = "";
-            $oUser = User::model()->findByPk($instance->oOptions->{$attribute});
+            /* \User|null[] see mantis #19426 */
+            static $oStaticUsers = array();
```

Comment:
I think I recall this is a caopy of a code already built in master.
Still, as to think for the future, shouldn't this be something implemented inside the user model?
I think we handle it inside models for other cases, right?

Reply:
> I think I recall this is a copy of a code already built in master. Still, as to think for the future, shouldn't this be something implemented inside the user model? I think we handle it inside models for other cases, right?

Already merged in master : https://github.com/LimeSurvey/LimeSurvey/pull/3785#issuecomment-1983265701

Reply:
Let's leave it like this then